charts/localstack/templates/deployment.yaml

```diff
@@ -60,6 +60,7 @@ spec:
             httpGet:
               path: /_localstack/health
               port: {{ .Values.service.edgeService.name }}
+            initialDelaySeconds: 120
```

Comment:
Could you explain a bit why this would be necessary?
120 seconds is quite long, if we have to add the `initialDelaySeconds` for a deployment on EKS, we should add it in a parameterized way with a way shorter value (because for the vast majority of users, this setting will slow down their deployment by at least one minute).
When this has been added as a parameterized value, we could add it to the `eks-values.yaml` (as mentioned in the comment below).

Reply:
Needed this in EKS with Fargate as it takes so long to launch a new pod. I'll move to eks-values.yaml override.

Reply:
I agree with @alexrashed here, and the way this is constructed _always_ overrides what can be specified in an override file. See https://github.com/localstack/helm-charts/blob/ce47b1590605901650ab788556bc871efbd78b8d/charts/localstack/values.yaml#L97-L102

I also however agree the initial delay shouldn't be 0 seconds, as the container always takes _some_ time to start up, but the total 30 seconds (3 failure periods x a period of 10 seconds) is reasonably generous. When I try with EKS + fargate as you have been doing @cabeaulac, I can still get a working pod in 30 seconds, provided the pod resources are higher than the default. It feels to me that a bit of initial delay (e.g. 5 seconds) where failures to reach the health endpoint do not count towards a failing health check makes sense.